Dockerfile

```diff
@@ -12,27 +12,29 @@ LABEL version="1.3"
 # Software revision
 ENV REVISION="a4e066d559b8b7f377a8aba5ee7f4c05c5542b83"
 
-# Meaningful working paths
+# Web server root path
 ENV WWW_ROOT "/var/www/html/"
-ENV APP_FOLDER "php-phenomenal-portal-app-library"
+
+# App name as ENV variable
+ENV APP_NAME "php-phenomenal-portal-app-library"
 
 # Install required software
 WORKDIR ${WWW_ROOT}
 RUN apt-get update && apt-get install -y --no-install-recommends git python python-dev build-essential python-pip && \
-    git clone https://github.com/phnmnl/php-phenomenal-portal-app-library.git && \
-    git -C php-phenomenal-portal-app-library checkout $REVISION && \
-    cd ${APP_FOLDER}/bin/markdown2html && git submodule init && git submodule update && \
+    git clone https://github.com/phnmnl/${APP_NAME}.git && \
```

Comment:
Can you clone with `--depth 1`?

Reply:
Option `--depth 1` is incompatible with the strategy we are currently using to checkout a specific revision of the software